### ISSUE TITLE
SJS Initialization

### ISSUE THREAD

**Issue by mmcfarland** [WikiWatershed/model-my-watershed#1128]
I believe the post-sqldao access still has trouble initializing SJS correctly:

On staging at 007d1a328bd59ea2f69b0ec76298411fd1217ec9 - registered about 5 minutes after launch time of the worker instance.

```
2016-02-02 22:27:34,392 [JobServer-akka.actor.default-dispatcher-9] ERROR spark.jobserver.LocalContextSupervisorActor -  Exception after sending Initialize to jobManagerActor
akka.pattern.AskTimeoutException: Ask timed out on [Actor[akka://JobServer/user/context-supervisor/geoprocessing#251442010]] after [15000 ms]
        at akka.pattern.PromiseActorRef$$anonfun$1.apply$mcV$sp(AskSupport.scala:334)
        at akka.actor.Scheduler$$anon$7.run(Scheduler.scala:117)
        at scala.concurrent.Future$InternalCallbackExecutor$.scala$concurrent$Future$InternalCallbackExecutor$$unbatchedExecute(Future.scala:694)
        at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:691)
        at akka.actor.LightArrayRevolverScheduler$TaskHolder.executeTask(Scheduler.scala:467)
        at akka.actor.LightArrayRevolverScheduler$$anon$8.executeBucket$1(Scheduler.scala:419)
        at akka.actor.LightArrayRevolverScheduler$$anon$8.nextTick(Scheduler.scala:423)
        at akka.actor.LightArrayRevolverScheduler$$anon$8.run(Scheduler.scala:375)
        at java.lang.Thread.run(Thread.java:745)
2016-02-02 22:27:34,394 [JobServer-akka.actor.default-dispatcher-9] ERROR spark.jobserver.LocalContextSupervisorActor -  Unable to start context geoprocessing
akka.pattern.AskTimeoutException: Ask timed out on [Actor[akka://JobServer/user/context-supervisor/geoprocessing#251442010]] after [15000 ms]
        at akka.pattern.PromiseActorRef$$anonfun$1.apply$mcV$sp(AskSupport.scala:334)
        at akka.actor.Scheduler$$anon$7.run(Scheduler.scala:117)
        at scala.concurrent.Future$InternalCallbackExecutor$.scala$concurrent$Future$InternalCallbackExecutor$$unbatchedExecute(Future.scala:694)
        at scala.concurrent.Future$InternalCallbackExecutor$.execute(Future.scala:691)
        at akka.actor.LightArrayRevolverScheduler$TaskHolder.executeTask(Scheduler.scala:467)
        at akka.actor.LightArrayRevolverScheduler$$anon$8.executeBucket$1(Scheduler.scala:419)
        at akka.actor.LightArrayRevolverScheduler$$anon$8.nextTick(Scheduler.scala:423)
        at akka.actor.LightArrayRevolverScheduler$$anon$8.run(Scheduler.scala:375)
        at java.lang.Thread.run(Thread.java:745)
```

**Comment by mmcfarland** [comment]
For current (at the time of writing) worker: 10.0.3.78 - the services were restarted manually to test additional bugs.

**Comment by mmcfarland** [comment]
Addressed by #1133
